src/components/index/page-sections/StartupSectionContentCard.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from '@reach/router';
+import { FormattedMessage } from 'react-intl';
+
+const StartupSectionContentCard = ({ firm, description, link }) => {
```

Comment:
Finde es generell immer nice bei so strukturellen Komponenten einen 'className' als prop zur Verfügung zu stellen.

Reply:
Musste grad auch in Erfahrung bringen, dass es mehr als notwendig ist danke dir Lennart!